packages/utilities/tests/retry.test.ts

```diff
@@ -0,0 +1,26 @@
+import { retry } from '../src/lib/retry';
+
+describe('retry', () => {
+	test('GIVEN a simple string return THEN returns the same on first attempt', async () => {
+		const result = await retry(() => 'test', 3);
```

Comment:
If we use `vi.fn`, we can test how many times the function is called to ensure the amount is correct.